### PR TITLE
Lint Haml only for the new Bootstrap interface

### PR DESCRIPTION
**Pull request by dmarcoux**
As we discussed in the grooming meeting, we want to run haml-lint only for the new Bootstrap interface.

Please review every commit independently from others. It's easier to follow what changed.

### CODE REVIEW

File: src/api/.rubocop.yml [hunk 1]
```diff
@@ -25,12 +25,17 @@ Style/MultilineBlockChain:
...
+
 ##################### Metrics ##################################
 
 Metrics/BlockLength:
   Exclude:
     - 'config/routes.rb'
 
+# We use the same value in haml-lint
+Metrics/LineLength:
+  Max: 150
```
Comment (Ana06) [inline src/api/.rubocop.yml]: this is not having any effect: https://github.com/openSUSE/open-build-service/blob/af12274522c3864ccc4ed8824913aa7848b3aba0/src/api/.rubocop_todo.yml#L1120

It is not the first time this annoy me: https://github.com/rubocop-hq/rubocop/issues/4156 I wanted to solve it (https://github.com/rubocop-hq/rubocop/pull/5505) but it seems I didn't :see_no_evil:

Comment (dmarcoux) [reply src/api/.rubocop.yml]: True, it's annoying... I won't set it up then...

File: src/api/.rubocop.yml [hunk 2]
```diff
@@ -25,6 +25,7 @@ Style/MultilineBlockChain:
   Exclude:
     - 'spec/**/*'
 
+
```
Comment (bgeuken) [inline src/api/.rubocop.yml]: Was this intended?

Comment (dmarcoux) [reply src/api/.rubocop.yml]: Nope, good catch.

**Comment by dmarcoux** [comment]
Please @Ana06, have another look.

**Pull request merged**